PeerioTechnologies/peerio-mobile
# feat: unshare volume from dm [branch ch3313]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/3313/mobile-unshare-a-folder-in-a-dm-from-the-button
#### Testing instructions  
Share a folder from User A to User B from files list
Go to the DM with User B
Invoke the action sheet from the shared folder and click unshare
UX should give feedback that the folder has been unshared
Note on UX feedback: The user who was removed from the volume will not see the feedback "Folder was unshared" in chat until the app is restarted. I couldn't find a workaround but maybe @seavan can, the reason is that React doesn't detect a change in the props of the folder object.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
